notebooks/official/tensorboard/vertex_tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,856 @@
+{
```

Comment:
<p>Update the links to the notebook file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I think the paths are correct. Since we have not submitted it, we have 404s.</p>